### DISCUSSION TITLE
GSOD Volunteering

### DISCUSSION THREAD

**Discussion by chinyerenwalie** [wagtail/wagtail#10308] [General]
Hi Wagtail,

I am interested in volunteering for your organization for GSOD. 
How do I submit my request, and what's the appropriate email address?

**Comment by zerolab** [comment]
👋

Please see the https://wagtail.org/gsod/ information page. Here is the link to the expression of interest form in particular: https://wagtail.org/gsod-apply/. Head to [#gsod](https://wagtailcms.slack.com/archives/C051ERV9F9Q) if you want to ask any questions.
